packages/cli/src/ui/App.tsx

```diff
@@ -592,9 +592,9 @@ const App = ({ config, settings, startupWarnings = [], version }: AppProps) => {
     (result: IdeIntegrationNudgeResult) => {
       if (result.userSelection === 'yes') {
         if (result.isExtensionPreInstalled) {
-          handleSlashCommand('/ide enable');
+          handleSlashCommand('/ide enable', new AbortController().signal);
```

Comment:
Do we have a app-level abort signal so if the user kills the app it gets triggered? Ultimately everything should be cancellable at some layer. OR at the very least this indicates that the `/ide` commands can't actually be interacted with via `esc` /cc @skeshive